config/deploy.rb

```diff
@@ -47,5 +48,3 @@
 
 # update shared_configs before restarting app
 before 'deploy:publishing', 'shared_configs:update'
-
-after 'deploy:publishing', 'resque:pool:hot_swap'
```

Comment:
huh - given that this doesn't happen at the same step for everything ... is it wise to have it happen magically from dlss-capistrano??

Reply:
@ndushay let's hold up on the robot suites. looking into whether this is a meaningful change or not, testing in QA